botfront/imports/ui/components/stories/StoryFooter.jsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Segment, Breadcrumb, Icon } from 'semantic-ui-react';
+
+import './style.import.less';
+
+import StoryPathPopup from './StoryPathPopup.jsx';
+
+class StoryFooter extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {};
+    }
+
+    renderPath = () => {
+        const { storyPath } = this.props;
+        let pathArray = storyPath.split('__');
+        const pathBreadcrumbs = [];
+        const maxLength = 5;
+
+        if (pathArray.length > 3) {
+            pathBreadcrumbs.push();
+            pathArray = pathArray.slice(pathArray.length - maxLength, pathArray.length);
+            pathBreadcrumbs.push(
+                <StoryPathPopup
```

Comment:
Any JSX element generated iteratively needs a key prop. (You'll see the warnings when you open a console)